code/modules/mob/living/carbon/xenomorph/xenoprocs.dm

```diff
@@ -1,3 +1,22 @@
+/mob/living/carbon/xenomorph/verb/xeno_health_alerts()
+	set name = "Toggle Xeno Health Alerts"
+	set desc = "Toggles alerts when a low HP xeno is injured."
+	set category = "Alien"
+
+	toggle_xeno_health_alerts(src)
+
+/proc/toggle_xeno_health_alerts(mob/living/carbon/xenomorph/user)
```

Comment:
This should be put into the players savefile or onto the database so it is persistent between rounds.

Do make that into commented code so we can test without breaking stuff

Reply:
I added it as a client preference.